chapter_12_organising_test_files.asciidoc

```diff
@@ -652,12 +648,6 @@ through what happened:
     and inside the body of the `lambda` function,
     and tells us that it was attempt to find the `.invalid-feedback` element
     that failed.
-// CSANAD: I changed two things here:
```

Comment:
I added these comments along with my changes while both reviewing and updating chapter 12. Since these changes have already been merged, the explanations are no longer needed.